app/services/gpo_reminder_sender.rb

```diff
@@ -22,17 +21,21 @@ def send_emails(for_letters_sent_before)
 
   private
 
-  def profiles_due_for_reminder(letter_eligible_range)
+  def profiles_due_for_reminder(for_letters_sent_before)
     ActiveRecord::Base.transaction do
       ActiveRecord::Base.connection.execute(
         "SET LOCAL statement_timeout = #{LOCAL_DATABASE_TIMEOUT}",
       )
 
+      profile_eligible_range =
+        (IdentityConfig.store.usps_confirmation_max_days +
+          IdentityConfig.store.gpo_max_profile_age_to_send_letter_in_days).
+          days.ago..for_letters_sent_before
       Profile.joins(:gpo_confirmation_codes).
         where(
-          gpo_verification_pending_at: letter_eligible_range,
+          gpo_verification_pending_at: profile_eligible_range,
           gpo_confirmation_codes: { reminder_sent_at: nil },
-          deactivation_reason: [nil, :in_person_verification_pending],
```

Comment:
I thought this was here because a user might have a GPO letter for in-person proofing; is this not possible?

Reply:
Sonia confirmed that that flag is deprecated and no longer used.